### PR TITLE
Fixed the typo in confirmationblock and done with get and post reques…

### PR DESCRIPTION
**Pull request by nishp77**
Left with validator api ☺☺ #13

### CODE REVIEW

**Comment by nishp77** [comment]
Please don't merge yet

**Comment by Mirch** [comment]
> Please don't merge yet

You can use WIP in your PR name to prevent merging

File: src/Thenewboston.Tests/Bank/Api/BankConfirmationBlockServiceTests.cs [hunk 1]
```diff
@@ -143,10 +143,10 @@ private IBankConfirmationBlockService BuildConfirmationBlockPostMock()
         #region Tests
 
         [Fact]
-        public async void BankConfiramtionBlockReturnedAsync()
+        public async void BankConfirmationBlockReturnedAsync()
         {
             var service = BuildConfirmationBlockGetMock();
-            var returnedBankConfirmationBlock = await service.GetAllBankConfiramtionBlocksAsync();
+            var returnedBankConfirmationBlock = await service.GetAllBankConfirmationBlocksAsync();
```
Comment (joshman1019) [inline src/Thenewboston.Tests/Bank/Api/BankConfirmationBlockServiceTests.cs:149]: I had the hardest time typing that word... 😂

**Comment by nishp77** [comment]
guys would like to talk about this class of BankBlock under bankapi/model, this a response model not the bank block so I just renamed the file as of now to BankBlockResponse but not the class name as it was related to some other files

**Comment by Mirch** [comment]
There are some conflicts here :)

**Review by Mirch — APPROVED** [review]

**Pull request merged**